src/mscorlib/src/System/Collections/Generic/Dictionary.cs

```diff
@@ -129,6 +129,24 @@ private struct Entry {
             }
         }
 
+        public Dictionary(IEnumerable<KeyValuePair<TKey, TValue>> enumerable):
```

Comment:
`ConcurrentDictionary` uses `collection` as the parameter name instead of `enumerable` for its similar constructor. Same with `List<T>`'s constructor that takes an `IEnumerable<T>`. It'd be nice to be consistent.